uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/init.js

```diff
@@ -1,10 +1,17 @@
 'use strict';
+var config = require('../site-config');
 
 module.exports = function (casper, scenario) {
-  var config = require('../site-config');
   var loginFormSelector = 'form#user-login-form';
   var credentials = config.credentials[scenario.credential];
 
+  casper.echo('--------------------------------------------', 'COMMENT');
+  casper.echo('Running Scenario ' + scenario.count + ' : ' + scenario.label, 'PARAMETER');
```

Comment:
The formatting should be improved, you are putting the counter before the name of the scenario. Change it to

```bash
Running scenario "Contact Summary" (2 of 124)
```